OpenRA.Mods.Cnc/Traits/Buildings/ClonesProducedUnits.cs

```diff
@@ -39,7 +39,7 @@ public ClonesProducedUnits(ActorInitializer init, ClonesProducedUnitsInfo info)
 			faction = init.Contains<FactionInit>() ? init.Get<FactionInit, string>() : init.Self.Owner.Faction.InternalName;
 		}
 
-		public void UnitProducedByOther(Actor self, Actor producer, Actor produced)
+		public void UnitProducedByOther(Actor self, Actor producer, Actor produced, string type)
```

Comment:
Can you please work through the various files and change this from `type` to `productionType` to make it clearer what these are?